templates/campaigns_newsletter.php

```diff
@@ -0,0 +1,68 @@
+<?php 
+	include("{$theme_directory}/countries.php");
+	include("{$theme_directory}/languages.php");
+	$user = wp_get_current_user();
+	$user_meta = get_user_meta($user->ID, 'community-meta-fields', true);
+	$user_language = is_array($user_meta) && sizeof($user_meta) > 0 && isset($user_meta['languages']) && sizeof($user_meta['languages']) > 0 ? $user_meta['languages'][0] : '';
+	$user_country = is_array($user_meta) && sizeof($user_meta) > 0 && isset($user_meta['country']) ? $user_meta['country'] : '';
+?>
+
+<div class="content newsletter__container">
+	<div class="row">
+		<div class="col-lg-6 newsletter__details">
+			<h2>Get Updates</h2>
+			<p class="newsletter__text">Subscribe to our newsletter and join Mozillians all around the world and learn about impactful opportunities to support Mozilla’s mission.</p>
+		</div>
+		<div class="col-lg-6 newsletter__signup">
+			<form id="newsletter_form" name="newsletter__form" action="https://www.mozilla.org/en-US/newsletter/" method="post" class="newsletter__form" novalidate>
+				<input type="hidden" id="fmt" name="fmt" value="H">
+				<input type="hidden" id="newsletters" name="newsletters" value="about-mozilla">
+				<div id="newsletter_email" class="newsletter__fields">
+					<label class="newsletter__label" for="newsletter-email">Email</label>
+					<input class="newsletter__input" aria-label="Enter your e-mail" aria-required="true" type="email" id="newsletter-email" name="email" required="" value="<?php echo (isset($user->user_email) && strlen($user->user_email) > 0 ? $user->user_email : '')?>">
+					<p class="newsletter__error">Invalid email.</p>
+				</div>
+				<div id="newsletter_details">
+					<div class="newsletter__fields">
+						<label class="newsletter__label" for="newsletter-country">Country or Region</label>
+						<select class="newsletter__dropdown" id="newsletter-country" name="country">
```

Comment:
This is causing issues on the profile page.  There is already a select element with name country.  You will need to change this as it's causing the save profile to not save the country for the profile.